tmcprototype/CspSubarrayLeafNode/CspSubarrayLeafNode/CspSubarrayLeafNode.py

```diff
@@ -316,17 +316,56 @@ def ConfigureScan(self, argin):
         # PROTECTED REGION END #    //  CspSubarrayLeafNode.ConfigureScan
 
     @command(
-        dtype_in='str',
+    dtype_in=('str',), 
     )
     @DebugIt()
     def StartScan(self, argin):
         # PROTECTED REGION ID(CspSubarrayLeafNode.StartScan) ENABLED START #
         """
-        This command starts the scan.
+        This command invokes Scan command on CspSubarray. This command is allowed only when CspSubarray is in
+        READY state.
 
-        :param argin:
-        :return:
+        :param argin: Timestamp
```

Comment:
It is not mentioned here because from SubarrayNode scanDuration is passed to CspSubarrayLeafNode. Currently, "0" is passed as an argument to ConfigureScan command of CspSubarray. So no where Timestamp is used.